src/solver/simulation/timeseries-numbers.cpp

```diff
@@ -716,14 +597,11 @@ void drawAndStoreTSnumbersForNOTintraModal(const array<bool, timeSeriesCount>& i
     for (auto& group: study.bindingConstraintsGroups)
     {
         const auto nbTimeSeries = group->numberOfTimeseries();
-        auto& value = group->timeseriesNumbers[0][year];
-        if (nbTimeSeries == 1)
+        auto& groupTsNumber = group->timeseriesNumbers[0][year];
+        groupTsNumber = 0;
+        if (nbTimeSeries != 1)
```

Comment:
```suggestion
        if (nbTimeSeries == 1)
                groupTsNumber = 0;
        else
                // ...
```

Reply:
Now 1-d Matrix **timeseriesNumbers** is reset (it's given zeros everywhere) as soon as it is sized, so we don't need to assign it a zero when associated TS matrix is one column sized.